RandBLAS/skge.hh

```diff
@@ -596,6 +596,84 @@ void sketch_general(
     return sketch_general(layout, opA, opS, m, d, n, alpha, A, lda, S, 0, 0, beta, B, ldb);
 };
 
+// =============================================================================
+/// \fn sketch_vector(blas::Op opS, int64_t d, int64_t m, T alpha, SKOP &S,
+///    int64_t i_off, int64_t j_off, const T *x, incx, T beta, T *y, incy
+/// )
```

Comment:
As a convenience function, I think we should offer ``sketch_vector`` that omits ``i_off`` and ``j_off`` and just sets them to zero. You don't need to test this function separately. The documentation can be copied from what you end up writing here (make sure we agree on it first!), and then you can trim down the copied text.